src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -68,14 +68,14 @@ spec:
 
 No matter if you are using Helm Charts or GitOps and Kustomize, just put the ServiceMonitor CR next to your app and apply it in the same way. Once it's applied you can check either the _ServiceMonitors Overview_ or _ServiceMonitors Details_ dashboards, or just search for the new metrics ingested in your installation's Grafana to make sure that your containers are being scraped by the new monitoring agents.
 
-__Warning:__ The ServiceMonitor needs to be labeled with `application.giantswarm.io/team: <YOUR-TEAM-NAME>` for the metrics agent to be able to discover it and start collecting metrics.
+__Warning:__ The ServiceMonitor needs to be labeled with `observability.giantswarm.io/tenant: <YOUR-TENANT-NAME>` for the metrics agent to be able to discover it and start collecting metrics.
```

Comment:
We also need a warning to explain that the tenant needs to be in a grafana organization right?

Reply:
Good point

Reply:
That's done. WDYT ?